### PR TITLE
VAV CANONICAL TYPE RUNNING ON OCCUPANCY SENSOR

### PR DESCRIPTION
**Pull request by darshanjain12**
VAV Canonical type which are pressure independent VAVs having high and low limits, setpoint based on occupancy.

### CODE REVIEW

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -657,6 +657,7 @@ literals:
 - heat_exchange_supply_water_isolation_valve_percentage_sensor
 - heat_exchange_return_water_isolation_valve_percentage_sensor
 
+
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:660]: please remove extra line

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -688,6 +688,18 @@ HSOUC:
...
 
+CSOUC:
+  description: "Cooling occupied/unoccupied setpoint control."
+  is_abstract: true
+  uses:
+  - zone_air_unoccupied_cooling_temperature_setpoint
+  - zone_air_occupied_cooling_temperature_setpoint
+  - zone_air_temperature_sensor
+  opt_uses:
+  - zone_air_temperature_setpoint
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:699]: How can a unit have this and the specific cooling setpoints?

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -4656,6 +4668,19 @@ HLSAFS:
   implements:
   - CONTROL
 
+HLDCAFS:
+  description: " VAV type with high and low limit Cooling setpoints"
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4672]: Please make this more general (remove VAV) and make the "C" in cooling lowercase.

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 4]
```diff
@@ -622,7 +622,7 @@ literals:
 
 # Added 1/16/2020
 - discharge_air_damper_percentage_command
-
+- discharge_air_damper_percentage_sensor
```
Comment (cstirdivant) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:625]: Please clarify the location of this damper on the VAV.

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 3 above)
Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4672]: Found a proper canonical type closing the request

**Pull request closed without merging**